### PR TITLE
add postgres 15

### PR DESCRIPTION
**Pull request by szaimen**
After https://github.com/nextcloud/server/pull/34645

Signed-off-by: Simon L <szaimen@e.mail.de>

### CODE REVIEW

File: admin_manual/installation/system_requirements.rst [hunk 1]
```diff
@@ -24,7 +24,7 @@ For best performance, stability and functionality we have documented some recomm
 +------------------+-----------------------------------------------------------------------+
 | Database         | - **MySQL 8.0+ or MariaDB 10.2/10.3/10.4/10.5** (recommended)         |
 |                  | - Oracle Database 11g (*only as part of an enterprise subscription*)  |
-|                  | - PostgreSQL 10/11/12/13/14                                           |
+|                  | - PostgreSQL 10/11/12/13/14/15                                           |
```
Comment (nickvergessen) [inline admin_manual/installation/system_requirements.rst:27]:
```suggestion
|                  | - PostgreSQL 10/11/12/13/14/15                                        |
```

Comment (nickvergessen) [reply admin_manual/installation/system_requirements.rst:27]: (layout of the markdown table is messed up (you need to remove 3 spaces)

Comment (szaimen) [reply admin_manual/installation/system_requirements.rst]: done

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**